### PR TITLE
Update routerlicious dependencies to use any binary blob on FirstSummary.

### PR DESCRIPTION
**Pull request by NicholasCouri**
## Description

Once the FRS server has been updated, we will need to start consuming the new base64 Blob on first summary. This PR does the client update.
[Support upload of any binary content of the blob during the initial summary upload at routerlicious driver. · Issue #15969 · microsoft/FluidFramework (github.com)](https://github.com/microsoft/FluidFramework/issues/15969)

Pull Requests that prepared for this change: 
[Server: Add infrastructure to support any type of Blob by NicholasCouri · Pull Request #16286 · microsoft/FluidFramework (github.com)](https://github.com/microsoft/FluidFramework/pull/16286/files)

[R11S, T9S: Consume convertFirstSummaryWholeSummaryTreeToSummaryTree to support all types of Blobs by NicholasCouri · Pull Request #16397 · microsoft/FluidFramework (github.com)](https://github.com/microsoft/FluidFramework/pull/16397)

### CODE REVIEW

**Review by tyler-cai-microsoft — APPROVED** [review]

File: packages/drivers/routerlicious-driver/src/documentServiceFactory.ts [hunk 1]
```diff
@@ -150,7 +152,12 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
 					await ordererRestWrapper.post<
 						{ id: string; token?: string; session?: ISession } | string
 					>(`/documents/${tenantId}`, {
-						summary: convertSummaryToCreateNewSummary(appSummary),
+						summary: convertSummaryTreeToWholeSummaryTree(
+							undefined,
+							appSummary,
+							"",
```
Comment (andre4i) [inline packages/drivers/routerlicious-driver/src/documentServiceFactory.ts:158]: Can the last two parameters be omitted? These are already the default values..

**Review by andre4i — APPROVED** [review]

**Comment by tylerbutler** [comment]
@NicholasCouri sorry for the dumb questions... do you know if FRS has already picked up the server changes? And this change will require client consumers to upgrade their server bits too, right? If so, add a changeset which will end up the release notes.

**Comment by tylerbutler** [comment]
@NicholasCouri Also can you add links to the relevant PRs that made the preceding changes? That will be really helpful for the commit if we need to trace the changes altogether.

**Comment by NicholasCouri** [comment]
> @NicholasCouri Also can you add links to the relevant PRs that made the preceding changes? That will be really helpful for the commit if we need to trace the changes altogether.

Done! Added to the description.

**Comment by NicholasCouri** [comment]
> @NicholasCouri Also can you add links to the relevant PRs that made the preceding changes? That will be really helpful for the commit if we need to trace the changes altogether.

Done.

File: .changeset/dry-windows-invent.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/routerlicious-driver": major
+---
+
+We are updating the client to adjust the behavior of the routerlicious driver during the first summary, which will now allow non-UTF-8 compatible binaries to be submitted. To support this change, it is necessary for the servers to run the latest versions that are prepared to work with this new format.
+
```
Comment (tylerbutler) [inline .changeset/dry-windows-invent.md:6]:
```suggestion
Server compatibility change: Upload first summary as base64 blob

We are updating the client to adjust the behavior of the routerlicious driver during the first summary, which will now allow non-UTF-8 compatible binaries to be submitted. (See [PR #16286](https://github.com/microsoft/FluidFramework/pull/16286) and [PR #16397](https://github.com/microsoft/FluidFramework/pull/16397)). To support this change, it is necessary for the servers to run the latest versions that are prepared to work with this new format.

This means that this version of routerlicious-driver requires routerlicious server version >=1.0.0.

```

**Review by tylerbutler — COMMENTED** [review]
Minor changeset tweaks.

**Comment by NicholasCouri** [comment]
[celebrate]     Nicholas Couri reacted to your message:
________________________________

**Pull request merged**